### ISSUE TITLE
Reduce power consumption on ESP32

### ISSUE THREAD

**Issue by JshStadler** [wernerhp/esphome.centurion_d5_evo#2]
The below can be added to the YAML for the ESP to reduce the clock speed of the ESP32 to lower power consumption. It's the lowest speed that supports Wi-Fi, and it works perfectly for me:

```
esphome:
  platformio_options:
    board_build.f_cpu: 80000000L
```
I'm not sure how to share this here without logging an "issue".

**Comment by wernerhp** [comment]
How much does it reduce the power by? Is it a significant amount?  Do you have any benchmarks?

**Comment by JshStadler** [comment]
@wernerhp 
Not exactly, but it does make a slight difference. And just for controlling the gate it doesn't need the extra speed.
Below are some indications, but I can't confirm how accurate they are.

https://esp32.com/viewtopic.php?t=2662#:~:text=At%2080%20MHz%2C%20with%20two,one%20CPU%20it%20consumes%2040mA.

![image](https://github.com/wernerhp/esphome.centurion_d5_evo/assets/37085670/c3cfdf70-8232-4e0e-b425-6863e08b28d5)
